telegraph/api.py

```diff
@@ -109,14 +109,19 @@ def edit_account_info(self, short_name=None, author_name=None,
             'author_url': author_url
         })
 
-    def revoke_access_token(self):
+    def revoke_access_token(self, replace_token=None):
         """ Revoke access_token and generate a new one, for example,
             if the user would like to reset all connected sessions, or
             you have reasons to believe the token was compromised.
             On success, returns dict with new access_token and auth_url fields
         """
 
-        return self._telegraph.method('revokeAccessToken')
+        response = self._telegraph.method('revokeAccessToken')
+        
+        if replace_token:
+            self._telegraph.access_token = response.get('access_token')
```

Comment:
hmm, I think `replace_token` arg is not really needed here

Reply:
If you agree with replacing access_token, replacing can be automated.

Reply:
Yeah, I think we can just replace current token, since it gets invalid anyway